index.ts

```diff
@@ -24,11 +24,11 @@ async function main () {
   ditto.store.registerObserver(`
     SELECT *
     FROM tasks
-    WHERE isDeleted = false`,
+    WHERE isDeleted = false AND isCompleted = false`,
     (result) => {
-      result.items
+       result.items
         .map((doc) => {
-          return doc.value
+          tasks.push(doc.value)
```

Comment:
I think we don't want `tasks.push` but instead `tasks = result.items.map()..)` because each time the observer fires, it fires with the full state of the database query

Reply:
Ah, got it. Ok I changed it as you suggested.